versions/build.s4.ver

```diff
@@ -0,0 +1,33 @@
+export hpc_ver=1.1.0
+export hpc_intel_ver=18.0.4
+export hpc_impi_ver=18.0.4
+
+export cmake_ver=3.20.1
```

Comment:
There isn't a module for cmake on S4.  The native version is 3.20.5 if it is desired to keep version numbers..

Reply:
Changed to `3.20.5` for now (done @ f1322477). All build.*.ver files will need full reviews ahead of them being used later. Thanks Dave!